pkgs/by-name/gu/guile-websocket/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  guile,
+  guile-gnutls,
+  texinfo,
+  pkg-config,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-websocket";
+  version = "0.2.0";
+
+  src = fetchurl {
+    url = "https://files.dthompson.us/releases/guile-websocket/${pname}-${version}.tar.gz";
+    sha256 = "sha256-7jxj+I5WpqtGu99zrzl92eIZUThy69A4CsLzXnp4dpA=";
+  };
+
+  strictDeps = true;
+  nativeBuildInputs = [
+    guile
+    pkg-config
+    texinfo
+  ];
+  buildInputs = [
+    guile
+    guile-gnutls
+  ];
+  doCheck = true;
+  makeFlags = [ "GUILE_AUTO_COMPILE=0" ];
+
+  meta = with lib; {
+    description = "guile-websocket provides an implementation of the WebSocket protocol.";
```

Comment:
would there happen to be a changelog?

Reply:
I looked at their website and the git file tree, unfortunately no changelog is available AFAIK.